cloud/hack/test-integration.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(unset CDPATH && cd $(dirname "${BASH_SOURCE[0]}")/../../ && pwd)
+export KUBE_CONFIG=""
+export KUBE_APISERVER_URL="http://localhost:8080"
+
+runTests() {
+
+  cleanup
+
+  logStatus "Applying device crd yamls"
+  kubectl create -f "${SCRIPT_ROOT}/build/crds/devices/devices_v1alpha1_devicemodel.yaml"
+  kubectl create -f "${SCRIPT_ROOT}/build/crds/devices/devices_v1alpha1_device.yaml"
+
+  logStatus  "Running integration test cases"
+  export KUBE_RACE="-race"
+  make -C "${SCRIPT_ROOT}/cloud" test \
+      WHAT="${WHAT:-$(findIntegrationTestDirs | paste -sd' ' -)}" \
+      GOFLAGS="${GOFLAGS:-}" \
+      KUBE_TEST_ARGS="--alsologtostderr=true ${KUBE_TEST_ARGS:-} " \
+      KUBE_RACE="" \
+
+  cleanup
+}
+
+findIntegrationTestDirs() {
+
```

Comment:
could we delete these empty lines in the funcs?

Reply:
done. thanks.